### DISCUSSION TITLE
Unable to connect with Jitsi server in SEB windows

### DISCUSSION THREAD

**Discussion by ganeshkumar-sr** [SafeExamBrowser/seb-win-refactoring#149] [Q&A]
I have created a configuration file with Jitsi features in ios SEB browser, We can't proctor the students in Jitsi meet(https://meet.jit.si/), when that configuration file opened in Windows SEB(3.2.0). The student can access the starturl and camera also turned on, but the student not joined in the Jitsi room.The same configuration works when we open it ios mobile, we can see the students in Jitsi meet room.

**Comment by dbuechel** [comment]
The proctoring implementation with Jitsi Meet is still experimental as of version 3.3.0. Live proctoring with Zoom is now fully implemented and ready to use, but Jitsi Meet won't be so at least until the upcoming version 3.4.0. Having said that, could you please share the log files for the affected session(s)? They can be found under `%LocalAppData%\SafeExamBrowser\Logs`.

**Reply by ganeshkumar-sr** [reply]
Please find the logs for an affected session in the attachment and assist us.The same file works perfectly in my ios mobile.
[2021-08-07_07h33m32s_Runtime.log](https://github.com/SafeExamBrowser/seb-win-refactoring/files/6948466/2021-08-07_07h33m32s_Runtime.log)
[2021-08-07_07h33m32s_Client.log](https://github.com/SafeExamBrowser/seb-win-refactoring/files/6948467/2021-08-07_07h33m32s_Client.log)
[2021-08-07_07h33m32s_Browser.log](https://github.com/SafeExamBrowser/seb-win-refactoring/files/6948468/2021-08-07_07h33m32s_Browser.log)

**Reply by ganeshkumar-sr** [reply]
@dbuechel It would be really helpful if you share any document to enable remote proctoring with Zoom or any example config file to look into.Thanks.

**Reply by vanie76** [reply]
@ganeshkumar-sr
I have the sample config file. I can email it to you
It works both for MAC and Windows.

**Reply by ganeshkumar-sr** [reply]
@vanie76 Could you please send me the configuration file to my profile email address?

**Reply by nmsundar-coder** [reply]
Hi @dbuechel  @vanie76 

I am also facing similar situation with SEB + JITSI integrated with custom LMS platform instead of existing LMS like Moodle. Any help would be appreciable ?

**Comment by mbaddar** [comment]
I'm getting an error on SEB 3.3.2 on Windiws. Jitsi room is open and moderator is online, yet on SEB I get this error: "https's server IP address cannot be found"

I have a standalone Jitsi server and I can start meetings with a generated JWT tokens with no issues. I have tried to change configuration parameters and nothing seems to work. Any help would be highly appreciated

**Reply by dbuechel** [reply]
The proctoring implementation with Jitsi Meet is still experimental, and we currently don't have a schedule to revisit it. Once we get to it, we'll definitely investigate your issue.

**Comment by mbaddar** [comment]
I assume [issue 154](https://github.com/SafeExamBrowser/seb-win-refactoring/issues/154) should be done in 3.4.0 as it's on your TODO list. Am I missing something?

**Reply by dbuechel** [reply]
Yes indeed, that is correct, we'll definitely do that for version 3.4.0. However, I don't know whether there will be time to further improve the Jitsi Meet implementation.
